src/colorize.cr

```diff
@@ -124,9 +124,11 @@ module Colorize
   # ```
   class_property? enabled : Bool = true
 
-  # Makes `Colorize.enabled` `true` if and only if both of `STDOUT.tty?` and `STDERR.tty?` are `true`.
+  # Makes `Colorize.enabled` `true` if and only if both of `STDOUT.tty?`
+  # and `STDERR.tty?` are `true`.
+  # If `TERM=dumb`, color won't be enabled either.
```

Comment:
This needs to explain that `TERM` is an environment variable. You removed that part from my suggestion.

Reply:
when I accepted the suggestion the comments got all mangled, so I guessed what was meant to be there.